packages/react-next/src/components/Checkbox/examples/Checkbox.CustomStyle.Example.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { DefaultButton } from '@fluentui/react-next/lib/Button';
+import { Checkbox } from '@fluentui/react-next/lib/Checkbox';
+import { Stack } from '@fluentui/react-next/lib/Stack';
+import { Customizer } from '@fluentui/react-next/lib/Utilities';
+import { ThemeProvider, loadTheme, FontWeights, FontSizes, IPartialTheme } from '@fluentui/react-next/lib/Styling';
+
+// Used to add spacing between example checkboxes
+const stackTokens = { childrenGap: 10 };
+
+const customizedTheme: IPartialTheme = { fonts: { medium: { fontWeight: FontWeights.bold } } };
+
+// TODO (xgao): Remove. This example is temporary for testing backward compatibility.
+export const CheckboxCustomStyleExample: React.FunctionComponent = () => {
+  const loadCustomizedTheme = () => loadTheme({ fonts: { medium: { fontSize: FontSizes.xLarge } } });
+
+  return (
+    <ThemeProvider>
+      <Stack tokens={stackTokens}>
+        <Customizer settings={{ theme: customizedTheme }}>
+          <Checkbox label="With theme" />
+        </Customizer>
+
+        <Customizer scopedSettings={{ Checkbox: { theme: customizedTheme } }}>
+          <Checkbox label="With scoped theme (not working)" defaultChecked />
```

Comment:
not working?

Reply:
Nope. this is providing **theme**  (not styles) to component. As discussed, we should abort supporting this